recipes/simdjson/0.5.0/conanfile.py

```diff
@@ -0,0 +1,97 @@
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "threads": [True, False]}
+    default_options = {'shared': False,
+                       'fPIC': True,
+                       'threads': True}
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+          return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+          return "build_subfolder"
+
+    @property
+    def _supported_cppstd(self):
+        return ["11", "gnu11", "14", "gnu14", "17", "gnu17", "20", "gnu20"]
+
+    def _is_supported_compiler(self):
+        # Try to get by conan. We support more compilers than that.
+        supported_compilers = [("apple-clang", 10), ("gcc", 7.4), ("clang", 6), ("Visual Studio", 15.7)]
+        compiler, version = self.settings.compiler, Version(self.settings.compiler.version)
+        return any(compiler == sc[0] and version >= sc[1] for sc in supported_compilers)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd and \
+           not self.settings.compiler.cppstd in self._supported_cppstd:
+          raise ConanInvalidConfiguration("This library requires c++11 standard or higher."
+                                          " {} required."
+                                          .format(self.settings.compiler.cppstd))
+
+        if not self._is_supported_compiler():
+            raise ConanInvalidConfiguration("This library is tested with a family of recent compilers."
+                                            " {} {} is not supported."
+                                            .format(self.settings.compiler, self.settings.compiler.version))
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions['SIMDJSON_BUILD_STATIC'] = not self.options.shared
+        self._cmake.definitions['SIMDJSON_ENABLE_THREADS'] = self.options.threads
+        self._cmake.definitions['SIMDJSON_SANITIZE'] = False
+        self._cmake.definitions['SIMDJSON_JUST_LIBRARY'] = True
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, 'lib', 'cmake'))
+        tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
```

Comment:
- Can you please check the names of the generated `.pc` files? Is it only `simdjson.pc`?
- Can you also check the name of the exported cmake files? Is it `simdjsonConfig.cmake`? What is the name of the exported targets inside it? simdjson::simdjson?

Reply:
1. Seems like simdjson doesn't install `pkgconfig` at all. I removed unneded line.
2. Exported file by upstream is `simdjson-config.cmake`. Exported target is simdjson::simdjson. Default value for `cpp_info.name` fits well for autogenerated cmake config files.